src/main/resources/standardTestCases/invalidate_bad_caches/invalidate_bad_caches_no_copy.wdl

```diff
@@ -0,0 +1,50 @@
+task make_file {
+    Boolean ready = true
+    command {
+        echo woohoo > out.txt
+    }
+    runtime {
+        docker: "ubuntu@sha256:71cd81252a3563a03ad8daee81047b62ab5d892ebbfbf71cf53415f29c130950"
+        backend: "Jes-Caching-No-Copy"
+    }
+    output {
+        File out = "out.txt"
+    }
+}
+
+task read_file {
+    File input_file
+    command {
+      cat ${input_file}
+    }
+    runtime {
+        docker: "ubuntu:latest"
+        backend: "Jes-Caching-No-Copy"
+    }
+    output {
+        File out = stdout()
+    }
+}
+
+task delete_file_in_gcs {
+    String file_path
+    command {
+        gsutil rm ${file_path}
+    }
+    runtime {
+        docker: "google/cloud-sdk@sha256:fb904276e8a902ccd9564989d9222bdfbe37ffcd7f9989ca7e24b4019a9b4b6b"
+        backend: "Jes-Caching-No-Copy"
```

Comment:
If I understand correctly, the backend in the test file only says "run only this test if the cromwell you're pointing to has a backend called X", but it doesn't actually make the test run on this backend, it will run on whatever the default backend is OR on the backend overridden in the runtime attribute.
I might have it wrong though I'm not 100% sure how all this works